<?xml version="1.0" encoding="UTF-8"?>
<interface xmlns="http://xml.phpdox.net/src" full="Psr\Cache\CacheException" namespace="Psr\Cache" name="CacheException" abstract="false" final="false" start="8" end="10">
  <file path="//var/www/html/vendor/psr/cache/src" file="CacheException.php" realpath="/var/www/html/vendor/psr/cache/src/CacheException.php" size="143" time="2016-08-06T20:24:11+00:00" unixtime="1470515051" sha1="b038c248daa0bb0650ed2f2fb57037447de1c595" relative="ar/www/html/vendor/psr/cache/src/CacheException.php"/>
  <docblock>
    <description compact="Exception interface for all exceptions thrown by an Implementing Library."/>
  </docblock>
  <extenders>
    <extender full="Psr\Cache\InvalidArgumentException" namespace="Psr\Cache" name="InvalidArgumentException"/>
  </extenders>
  <implementors>
    <class full="Symfony\Component\Cache\Exception\CacheException" namespace="Symfony\Component\Cache\Exception" name="CacheException"/>
    <class full="Symfony\Component\Cache\Exception\LogicException" namespace="Symfony\Component\Cache\Exception" name="LogicException"/>
  </implementors>
</interface>
